From 7fc46ad965497c3b2f8b30f7a7f6af1016150aa7 Mon Sep 17 00:00:00 2001
From: Jinman Zhao <jinman.zhao@gmail.com>
Date: Mon, 6 Apr 2020 14:22:59 -0500
Subject: [PATCH] eval.py support case

---
 eval.py | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/eval.py b/eval.py
index 3dad904..9c5424d 100644
--- a/eval.py
+++ b/eval.py
@@ -46,6 +46,8 @@ parser.add_argument(
     required=True,
     help='path to data'
 )
+parser.add_argument('--lower', action='store_true', default=True)
+parser.add_argument('--no_lower', dest='lower', action='store_false')
 args = parser.parse_args()
 
 vectors = {}
@@ -73,8 +75,10 @@ nwords = 0.0
 fin = open(args.dataPath, 'rb')
 for line in fin:
     tline = compat_splitting(line)
-    word1 = tline[0].lower()
-    word2 = tline[1].lower()
+    word1 = tline[0]
+    word2 = tline[1]
+    if args.lower:
+        word1, word2 = word1.lower(), word2.lower()
     nwords = nwords + 1.0
 
     if (word1 in vectors) and (word2 in vectors):
-- 
2.17.1

